src/DynamoCore/Graph/Notes/NoteModel.cs

```diff
@@ -11,6 +11,14 @@ namespace Dynamo.Graph.Notes
     /// </summary>
     public class NoteModel : ModelBase
     {
+        /// <summary>
+        /// 
+        /// </summary>
+        public event Action<ModelBase> UndoRequest;
+
+        private bool createdPinNode;
```

Comment:
it seems you can just use auto properties here instead of the field.
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties


in addition /// summary tags are good, sometimes even for internal members.

Reply:
Fixed. Thanks!